doc/source/admin/cluster.md

```diff
@@ -216,6 +219,20 @@ If you need to add additional parameters to your condor submission, you can do s
 </destinations>
 ```
 
+The runner accepts the following optional parameters.
+
+``prefix``
+: A string that will be to prepended to HTCondor command line tool invocations. Defaults to an empty string.
```

Comment:
Make clear that the prefix is followed by the command __without__ any spaces between. So you need to add a trailing space to prefix if you want that.

Reply:
```suggestion
: A string that will be to prepended to HTCondor command line tool invocations. Defaults to an empty string. Note that
  the runner **does not include a space** between the prefix and the command, so most likely you want to add a space
  a the end of your `prefix`.
```

I will use this text then.